gcp_variant_transforms/beam_io/bgzf.py

```diff
@@ -153,13 +159,9 @@ class BGZFBlock(BGZF):
 
   It reads contents from `self._block`.
     - The string before first `\n` is discarded.
-    - Decompress the next 64KB after self._block and reads the first line. It
-      assumes one row can at most spans in two Blocks.
+    - Retrieves the next 16MB after self._block and reads the first line. It
```

Comment:
Why the change here from 64K to 16MB?

Reply:
I have updated the doc to explain where the 16 MB comes from.The bug occurs when the record spans more than one GZIP block (max size 64 KB). 16 MB is the default read size for compressed files. I think it should be safe enough to cover one record (in compressed form).